### DISCUSSION TITLE
Using the Occurrence Editor to suggest data cleaning changes to the record

### DISCUSSION THREAD

**Discussion by JRThorson** [Symbiota/community-board#462] [Q&A] [answered]
I have been researching the collections of D. C. Ingram (active from 1912 to 1929).  This has led me to several Symbiota-supported portals to search for and download records for his specimens.  Along the way, if I spotted an obvious error or mistype in the record, I would post a comment on the Detailed Collection Record Info page.

I also noticed that some (not all) collections allowed me to bring up the Occurrence Editor and make changes to the record, subject to review.  I am a new user to the site, and would not expect to have such editing privileges.  Is this a feature or a bug?

The Occurrence Editor form is well-laid out, and I could easily correct wrong field values.  It's a great way to submit suggestions to clean the data.  Some critical fields were editable, such as the occurrenceID.  I must assume that if I mistakenly changed such a field, the change would be caught and ignored.  So, is it proper for me to have editing privileges?  If so, are safeguards in place to prevent one from mashing up the record inadvertently?

**Comment by themerekat** [comment]
Hi @JRThorson , thanks for your feedback! Can you tell us which Symbiota portals you have editing access to?
If you're not affiliated with a collection or have an agreement with them, we recommend just making a comment on the specimen explaining the change that is needed.

**Comment by JRThorson** [comment]
I have been working from the swbiodiversity.org web site for the most
part.  Collections I came across which allow access to the Occurrence
Editor are NY, RM, RM-USFS and EIU.   A quick examination of other
collections brought up more cases of Occurrence Editor access.  The easiest
way to find such collections is to do a broad search, sort the table by
collection and examine a record from each collection.  The access appears
to be on a collection-wide basis.  Hope this helps.

Jeff

**Comment by themerekat** [comment, marked as answer]
@JRThorson , aha, yes those collections have the "Allow Public Edits" function enabled on their collections. This enables people like you to make edits to their specimen records that they can then approve or revert. This function is not super well-used at the moment, so it could be that these collections had it on from a previous project, or they are actively curating these public edits.
